dataprofiler/profilers/profile_builder.py

```diff
@@ -1257,7 +1260,8 @@ def report(self, report_options=None):
                 "unique_row_ratio": self._get_unique_row_ratio(),
                 "duplicate_row_count": self._get_duplicate_row_count(),
                 "file_type": self.file_type,
-                "encoding": self.encoding
+                "encoding": self.encoding,
+                "correlation_matrix": self.correlation_matrix
```

Comment:
Should either leave out of report for now or serialize it somehow (json or something of the sort)

Reply:
convert to json now